packages/website/sidebars.js

```diff
@@ -36,10 +36,6 @@ module.exports = {
           'getAlgoliaHits-js',
           'getAlgoliaResults-js',
           'getAlgoliaFacetHits-js',
-          'highlightHit',
-          'reverseHighlightHit',
-          'snippetHit',
-          'reverseSnippetHit',
```

Comment:
no new page for components was added to the sidebar here, expected?

Reply:
> We'll need to document this Component API in another PR.

I think it will be undocumented until then